### PR TITLE
Update ksonnet package for Katib

### PR DESCRIPTION
**Pull request by andreyvelich**
Fixes: https://github.com/kubeflow/kubeflow/issues/2126.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2135)

### CODE REVIEW

**Comment by andreyvelich** [comment]
@YujiOshima @johnugeorge @jlewi

File: kubeflow/katib/studyjobcontroller.libsonnet [hunk 1]
```diff
@@ -273,30 +273,6 @@
                   name: "studyjob-controller",
                   image: params.studyJobControllerImage,
                   imagePullPolicy: "Always",
-                  volumeMounts: [
-                    {
-                      name: "worker-template",
```
[the remark this answers is not in the document]
Comment (andreyvelich) [reply kubeflow/katib/studyjobcontroller.libsonnet]: @johnugeorge @YujiOshima said that we don't need it right now.
In this manifest we don't have this volume: 
https://github.com/kubeflow/katib/blob/master/manifests/studyjobcontroller/studyjobcontroller.yaml

[the remark this answers is not in the document]
Comment (johnugeorge) [reply kubeflow/katib/studyjobcontroller.libsonnet]: Thanks. I missed it. It directly fetches from config maps now

**Pull request merged**